### ISSUE TITLE
How to list ordinals (numbers) of utxo?

### ISSUE THREAD

**Issue by donnlee** [casey/ord#1684]
So a topped-up input has common sats appended to it in the output, iiuc.
How can I list all the ordinals (sat numbers) of a utxo? Is there a ord command?

Ref: issue #1538

**Comment by donnlee** [comment]
Seems it requires ord indexing with --index-sats
How big is this index?  >100GB?
Meanwhile, i am using the search box on ordinals.com

**Comment by raphjaph** [comment]
Yes build with `--index-sats` but you will need a lot of RAM and about 100gb of disk space.
